app.py

```diff
@@ -163,5 +165,37 @@ def setGridBoundries():
                               mimetype='application/json')
 
 
+@app.route('/export-data', methods=['POST'])
+def exportData():
+    if flask.request.is_json:
+        data = flask.request.get_json()
+
+        #TODO add/subtract/update information according to need
+        body = [{"studyName": data["flight_details"]["display_name"],
```

Comment:
study name cannot be the display name since it will be used for multiple plots. Change this to be a combination of the date, lead scientist and crop name
also, json keys should be snake case unless needed by brAPI

Reply:
Yes, this is the format needed by BrAPI

Reply:
> the date

By date, do you mean the mission_start_time's date or the current date?